Admin/Admin.php

```diff
@@ -2030,12 +2057,15 @@ public function getFilterFieldDescriptions()
     /**
      * {@inheritdoc}
      */
-    public function addChild(AdminInterface $child)
+    public function addChild(AdminInterface $child, $field = null)
```

Comment:
@Koc : It's not a BC-break for calling code, but it is a BC break for extending code. [See for yourself](https://3v4l.org/PDY3L)